### ISSUE TITLE
Add "Routing" option

### ISSUE THREAD

**Issue by PAYAMiN** [InvisibleManVPN/InvisibleMan-XRayClient#94]
Hi,
kindly add a "Routing" option same as Android/iOS apps so traffic can be managed (Direct or Proxy) according to IP or application.

**Comment by InvisibleManVPN** [comment]
Hi @PAYAMiN!
Thanks for submitting the issue. I'll check it and give you the feedback.

**Comment by MrRadrigos** [comment]
Yes, that would be an ideal feature. Or is it possible to specify in the configuration file which sites go through a proxy, and which ones go directly?

**Comment by LumineRose** [comment]
I support, very necessary function, if there was also an opportunity like hiddify to choose from the gui region and automatically do direct routing for example Ru region (geoip + domain), as well as make direct for bittorrent and games working on udp

**Comment by 2muaddlb** [comment]
+1
I tried different JSONs for routing (TUN), but still no success...
The program says to check config for errors.

Btw, [InvisibleManVPN](https://github.com/InvisibleManVPN), add error logging while one's trying to connect -- I'm getting none, when the error window pops up with "check config" message
